### DISCUSSION TITLE
识别英文的时候，对某些符号识别率低

### DISCUSSION THREAD

**Discussion by yanqiukong** [hiroi-sora/Umi-OCR#438] [General]
像常见的引号、省略号这些，有时候会识别不出来，如果某个段落完全是符号的话，可能就直接省略了，这个东西有办法优化吗

**Comment by hiroi-sora** [comment]
标签页设置 → 语言/模型库 → 不要用`简体中文`，调成 `English` ，可以提高识别率。

如果还不行的话，那就是技术受限了。一些符号可见度低，确实容易被省略。

**Reply by Neon2333** [reply]
大佬，这个调成English的过程，可以用命令行进行设置吗？

**Reply by cuicheng01** [reply]
试试PP-OCRv5吧，一个模型支持多种语言
